joomla/joomla-cms
# [4.0] Add tooltip for workflow list view text-search box #pbf19 #codepatch

### Summary of Changes
Adds tooltip to text-search box


### Testing Instructions
1. Apply patch
2. Verify if tooltip shown for search box


### Expected result
Tooltip shown for search box


### Actual result
Tooltip shown for search box


### Documentation Changes Required
No
